openfisca_uk/variables/finance/tax/income_tax/relief.py

```diff
@@ -54,6 +86,16 @@ class pension_contributions(Variable):
     label = u"Amount contributed to registered pension schemes paid by the individual (not the employer)"
     definition_period = YEAR
 
+    def formula(person, period, parameters):
+        return add(
+            person,
+            period,
+            (
```

Comment:
total nit but would be slightly nice to be consistent between lists and tuples as arg 3 to `add`

Reply:
Yeah you're right: should be tuples where lists are not needed